README.md

```diff
@@ -1,25 +1,50 @@
 # Convene
 
-Community-owned tools to bring people together, online and off.
+Space to (play || work || grow || be).
 
 ## Purpose
 
-Many public domain, open-source, and free software products leverage the [Open
-Core model] to be socioeconomically sustainable. The Open Core model allows
-producers and maintainers to capture a portion of the wealth generated by their
-efforts; but requires people or organizations to maintain their own
-infrastructure tooling or pay privately held companies to host the services they
-use.
+Convene is the _Means of Connection_. It's space to co-conspire, to relax, or
```

Comment:
Should it be `It's a space to co-conspire`?

Reply:
Stylistically, I think it works either way. `It's space to X` makes space the word that _defines_ `it` while `a space to X` makes it a word that _describes_ it.